cypress/integration/without-security/check_timeline.js

```diff
@@ -0,0 +1,119 @@
+/*
+ * Copyright OpenSearch Contributors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+/* eslint-disable import/no-unresolved */
+/* eslint-disable no-undef */
```

Comment:
Remind me why we need these in every file?

Reply:
Unfortunately Cypress isn't a dependency within this repo but during the execution of BWC tests it sets up Cypress. So the linter is failing and previously we had just disable the linter completely for each file. If you would like, I believ I can just modify `.eslintrc.js` to disable these rules for cypress/*.js

Reply:
Oh right. Yeah I'd say let's try to catch all of those with a change to the lint config. And then when we add Cypress in the future we can remove just one line.